doc/nrf/protocols/lte/index.rst

```diff
@@ -0,0 +1,214 @@
+Most cellular operators support both LTE-M and NB-IoT, but there are geographic locations where only one of these is supported.
+See `Mobile IoT Deployment Map`_ for more information.
+
+Certifications
+==============
+
+Some MNOs have their own certifications, like Verizon and Vodafone.
+But many operators only require GCF (Global Certification Forum) and PTCRB certifications that devices work as per standard, and regulatory certifications.
+
+For more information about certification related to the nRF91 Series, refer to the following pages:
+
+* `nRF91 Series certifications`_
+* `Certifying a Cellular IoT Device`_
+
+SIM card support
+================
+
+SIMs that are used with the nRF91 Series devices must have low-power WAN support and must be supported by the MNO.
+For more information on the iBasis SIM compatible with the nRF91 Series devices, refer to the `iBasis IoT network coverage`_ page.
+
+SoftSIM
+-------
+
+The nRF91 Series supports SoftSIM, which allows the usage of software SIM-based solutions to reduce energy consumption associated with physical SIMs.
+The following are some of the key advantages of using SoftSIM:
+
+* Cost savings if you remove the hardware.
+* Power savings in idle mode.
+  Applications using eDRX will benefit from using SoftSIM.
+* Increases battery life.
+
+See the SoftSIM support section of the `Cellular IoT unique features`_ documentation for information on SoftSIM.
+
+The SoftSIM support is available in |NCS| from the v2.5.0 release.
+The `Onomondo SoftSIM intergration with the nRF91 Series`_ guide describes the intergation of SoftSIM into |NCS|, and the :ref:`nrf_modem_softsim` guide demonstrates the transfer of SIM data between the modem and the application.
+
+Power consumption
+=================
+
+LTE-M and NB-IoT both support Power Saving Mode (PSM) or Extended Discontinuous Reception (eDRX) to minimize power consumption.
+For more information on power-saving techniques, refer to the DevAcademy's `Power saving techniques`_ documentation.
+
+Security
+========
+
+The nRF91 Series devices include a range of security features, such as Arm TrustZone and Arm CryptoCell for secure application and data handling.
+Security best practices are impletemented to protect data transmitted over the LTE network.
+This includes using encryption, secure boot, and secure firmware updates.
+
+Getting started with Cellular products
+**************************************
+
+For getting started on the cellular products, you can refer to the :ref:`ug_nrf91` documentation, which guides through the features, getting started steps, and advanced usage of the boards.
+For a self-paced, hands-on course, enroll in DevAcademy's `Cellular IoT Fundamentals course`_ , which describes the LTE and NB-IoT technologies and the software architecture of a cellular IoT application based on Nordic Semiconductor’s 91 Series cellular hardware.
```

Comment:
Please align the naming for `Nordic Semiconductor’s 91 Series cellular hardware`. I think the terms used in the documentation are:

- nRF91 Series device (nRF91 DKs and Thingy:91s)
- nRF91 Series SiP